crates/nu-protocol/src/value/range.rs

```diff
@@ -95,20 +95,28 @@ mod int_range {
 
         pub fn contains(&self, value: i64) -> bool {
             if self.step < 0 {
-                value <= self.start
-                    && match self.end {
-                        Bound::Included(end) => value >= end,
-                        Bound::Excluded(end) => value > end,
-                        Bound::Unbounded => true,
-                    }
+                // Decreasing range
```

Comment:
I could probably normalize the range (set start to the minimum value, end to the highest) and just apply this logic once